src/pages/professors.tsx

```diff
@@ -16,6 +17,14 @@ const Professors = ({ professors }) => {
         setDetailsIsOpen(!detailsIsOpen);
         setProf(prof);
     };
+    const { data, isLoading } = useGetQuery("/api/users");
```

Comment:
you'll need to do the same wrapper pattern as preferences to fix the build
unless there's another way somehow